app/scripts/app-init.js

```diff
@@ -111,14 +104,15 @@ function importAllScripts() {
   }
 }
 
-// eslint-disable-next-line no-undef
-self.addEventListener('install', importAllScripts);
+importAllScripts();
 
 /*
  * Message event listener below loads script if they are no longer available.
```

Comment:
Can we update this to @naugtur's [suggestion earlier](https://github.com/MetaMask/metamask-extension/pull/14781#discussion_r930724118)? I think the proposed comment provides a bit more context. 

```
A keepalive message receiver to avoid Service Worker getting shut down due to inactivity. UI sends the message periodically.
```